src/main/java/org/sagebionetworks/bridge/sdk/models/upload/UploadSchemaType.java

```diff
@@ -0,0 +1,14 @@
+package org.sagebionetworks.bridge.sdk.models.upload;
+
+/**
+ * Tags schemas with the schema type, so we can figure out how to parse the data into the specified fields. Currently,
+ * this only contains iOS-specific schema types. As we define new data formats, we will need to expand this enum. For
+ * more information, see https://sagebionetworks.jira.com/wiki/display/BRIDGE/Bridge+Upload+Data+Format
+ */
+public enum UploadSchemaType {
+    /** iOS data upload. For example, Parkinson's Tapping Activity. */
+    IOS_DATA,
+
+    /** iOS survey. For example, Asthma Weekly Prompt. */
+    IOS_SURVEY,
```

Comment:
It would be great to somehow get the clients back to submitting JSON we've defined for survey answers... we want everything in that payload. I take it right now this schema represents the JSON we're actually getting from the YML apps.